docs/database/db_schema-main.sql

```diff
@@ -59,6 +58,21 @@ CREATE TABLE `certificateStatus` (
   PRIMARY KEY (`serial`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+CREATE TABLE `challenges` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `authorizationID` char(43) NOT NULL,
```

Comment:
The authorization ID is a base64 encoded 32 byte string, so is always 43 characters long.